go-app-ussd_pmtct_rapidpro.js

```diff
@@ -169,14 +169,14 @@ go.app = function() {
         self.add("state_enter_msisdn", function(name) {
             return new FreeText(name, {
                 question:
-                    "Please enter the cell number of the mother who would like to sign up to " +
-                    "receive HIV-related messages from MomConnect e.g. 0813547654.",
+                    "Please enter the cell number of the mom that would like to sign up to or " +
```

Comment:
cell number changed to no. earlier, should that be happening here too?

Reply:
I did a copy and paste of how SxD did the screens, I could double check with them if this format was deliberate

Reply:
The change of the use of number to `no.` is used when there are character limitations, only 160 char can be used per screen. So its a decision they took as a standard across all of the USSD lines & message sets